andersoncd/tests/test_estimators.py

```diff
@@ -18,21 +18,26 @@
 dict_estimators_sk = {}
 dict_estimators_sk["Lasso"] = Lasso_sklearn(
     alpha=alpha, fit_intercept=False, tol=tol)
+dict_estimators_sk["wLasso"] = Lasso_sklearn(
+    alpha=alpha, fit_intercept=False, tol=tol)
 dict_estimators_sk["ElasticNet"] = ElasticNet_sklearn(
     alpha=alpha, l1_ratio=l1_ratio, fit_intercept=False, tol=tol)
 dict_estimators_sk["MCP"] = Lasso_sklearn(
     alpha=alpha, fit_intercept=False, tol=tol)
 
 dict_estimators_ours = {}
-dict_estimators_ours["Lasso"] = WeightedLasso(
```

Comment:
I would put similar estimators next to each other (alternating sk and us  (also us us instead oof ours, same length as sk hence it will align) )

Instead of the big sk block , followed by the us block

Reply:
to make it easier to check that parameters are the same